test/util.spec.js

```diff
@@ -113,9 +113,9 @@ describe('.util', () => {
   })
 
   it('.urlAdd http with redirection', (done) => {
-    ipfs.util.addFromURL('http://covers.openlibrary.org/book/id/969165.jpg', (err, result) => {
+    ipfs.util.addFromURL('https://coverartarchive.org/release/6e2a1694-d8b9-466a-aa33-b1077b2333c1', (err, result) => {
```

Comment:
@vmx can we use one file used from the ipfs gateway?

Reply:
If you give me a link, for sure.

I don't know how to find URLs that redirect. I wanted a quick solution to the problem. The real fix would be one that doesn't need an Internet connection at all, but that's out of scope for me atm.

Reply:
nvm. Missed the need for a redirect :)